lib/rules/lines-around-comment.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * @fileoverview Enforces empty lines around comments.
+ * @author Jamund Ferguson
+ * @copyright 2015 Jamund Ferguson. All rights reserved.
+ */
+"use strict";
+
+function getEmptyLines(context) {
+    var lines = context.getSourceLines();
+    var emptyLines = lines.map(function(line, i) {
+        return {
+            code: line.trim(),
+            num: i + 1
+        };
+    }).filter(function(line) {
+        return !line.code;
+    }).map(function(line) {
+        return line.num;
+    });
+    return emptyLines;
+}
+
+function getAllComments(context) {
```

Comment:
Same here:

1. Missing JSDoc comment
2. The function name indicates that it returns the content of the comment but it returns the line numbers.